apps/api/lib/helpers/captureErrors.ts

```diff
@@ -1,14 +1,20 @@
 import * as Sentry from "@sentry/nextjs";
 import type { NextMiddleware } from "next-api-middleware";
 
+import { redactError } from "@calcom/lib/redactError";
+
 export const captureErrors: NextMiddleware = async (_req, res, next) => {
   try {
     // Catch any errors that are thrown in remaining
     // middleware and the API route handler
     await next();
   } catch (error) {
     Sentry.captureException(error);
-    console.log(error);
+    const redactedError = redactError(error);
+    if (redactedError instanceof Error) {
```

Comment:
Is returning 400 (Bad Request) the correct error here? Are we able to properly return a 500 when it's an unexpected error or is it not possible at the moment based on how we are processing errors?